easybuild/easyblocks/n/nccl.py

```diff
@@ -57,6 +57,9 @@ def build_step(self):
         if nvcc_gencode:
             self.cfg.update('buildopts', 'NVCC_GENCODE="%s"' % ' '.join(nvcc_gencode))
 
+        # Set PREFIX to correctly generate nccl.pc
+        self.cfg.update('buildopts', "PREFIX=%s" % self.installdir)
```

Comment:
@branfosj can you add a sanity check that checks for this file?

Reply:
I've switched the sanity check in the easyconfig to test that this file exists: https://github.com/easybuilders/easybuild-easyconfigs/pull/12183/commits/3b08b68272e3e495b4ce6490433510420baa6042

Reply:
It's better to implement the sanity check in the easyblock though, and keep the easyconfig minimal, no?